src/patterns.yml

```
@@ -801,3 +824,4 @@ fin:
 - ", <plus_loin>"
 - ", <quelquepart>"
 - ", une bonne fois pour toutes"
+- ". Témoignage"
```

Est-ce qu’on a aussi “. Retour d’expérience” (ou “REXP” pour faire plus technique)

---

Ah ouais chaud ! Moins pour REXP parce que même moi il me faut 30 secondes pour me souvenir de ce que ça veut dire 😅 

```suggestion
- ". Témoignage"
- ". Retour d’expérience"
```